spec/team_builder_spec.rb

```diff
@@ -11,11 +11,11 @@
   let(:uri) { URI("https://docs.publishing.service.gov.uk/repos.json") }
 
   let(:repos) do
```

Comment:
It would be good to have some tests for a scenario when "team" is not the same as "alerts_team".

Reply:
I don't think we need any tests here, since govuk-developer-docs will always return a value for `alerts_team`.